cirq/google/__init__.py

```diff
@@ -30,6 +30,15 @@
     xmon_op_from_proto_dict,
 )
 
+from cirq.google.common_serializers import (
+    MEASUREMENT_DESERIALIZER,
```

Comment:
I wouldn't flatten these to the module level, as I wouldn't expect end users to use it.

Reply:
removed.